pyartcd/pyartcd/pipelines/quay_doomsday_backup.py

```diff
@@ -77,6 +76,12 @@ def run(self):
 @click.option("--version", required=True, help="Release to sync, e.g. 4.15.3")
 @pass_runtime
 def quay_doomsday_backup(runtime: Runtime, arches: str, all_arches: bool, version: str):
+
+    # In 4.12 we sync only x86_64 and s390x
+    if version.startswith("4.12"):
+        all_arches = False
+        arches = "x86_64,s390x"
+
     doomsday_pipeline = QuayDoomsdaySync(runtime=runtime,
                                          arches=arches,
```

Comment:
The type of `arches` in `QuayDoomsdaySync.__init__()` was wrong, it was supposed to be only `Optional[str]`.  This whole time we have been passing `str` (or `None`) to `QuayDoomsdaySync.__init__()` where we then split the `arches` into `list`. I must have missed this incorrect type in https://github.com/openshift-eng/art-tools/pull/997.
I've changed the type of `arches` to `Optional[str]`. Please let me know if you think it would be better to split `arches` before passing it to the `__init__` function (and as a result the type would be `Optional[list[str]]` again)